README.md

```diff
@@ -83,5 +83,69 @@ running the software, logging the errors and traffic, and sending us the
 results. If you are not comfortable emailing log files please contact us at
 support@activecountermeasures.com
 
-Please see [Adding Support For Additional Routers](docs/Router%20Support.md)
-for more information on gathering the data needed to get your device supported.
+Please see [Adding Support For Additional Routers](docs/Router%20Support.md) for more
+information on gathering the data needed to get your device supported.
+
+# Troubleshooting
+### Testing IPFix/Netflow Records
+To test that IPFix/Netflow records are arriving at you IPFIX-RITA system, run
+the following on the IPFIX-RITA system:
+```
+$ tcpdump -qtnp 'udp port 2055'
+```
+Though the actual IP addresses and length will be different, you should see
+lines like:
+```
+IP 10.0.0.5:2055 > 10.0.0.43:2055: UDP, length 212
+```
+arriving somewhat regularly. Press Ctrl-C to exit. If you don't get any of
+these lines after a minute or so, your router may not be configured correctly
+to send these records to the ipfix-rita system. Double check your router
+configuration; make sure it's sending records to the IPFIX-RITA system's IP
+address and to UDP port 2055.
+
+### Ensure Docker Containers are Running
+To make sure that all the docker containers are running correctly on the
+IPFIX-RITA system, run the following on that system:
+```
+sudo docker ps
+```
+You should get a header line starting with "CONTAINER ID" and then at least
+three lines of running ocntainers with a status on "Up (some amount of time)".
+The names of these containers should start with "ipfix_rita_logstash",
+"ipfix_rita_converter", and "ipfix_rita_mongodb". If you do not get these three
+lines, somehting may be wrong with the docker intances, please contact
+technical support.
+
+### Checking that IPFIX-RITA is Creating Mongo Databases
```

Comment:
You want to connect to whatever database is listed in the first connection uri in the config file. Most of the time, this will not be the same as the second connection uri in the config file. 

The first connection uri is used to store RITA compatible results.
The second connection uri is used to buffer incoming data. This uri is hardcoded into logstash.